src/Billing/Entities/RecurringCost.php

```diff
@@ -8,7 +8,7 @@
 
 /**
  * @property int $id
- * @property string $name
+ * @property Type $type
```

Comment:
We don't seem to have an Entity for `Type`. Can we add one please, @drewsshughes?

In the same way that you've done the `Product` Entity.

Reply:
As a result of adding this, we need to make sure the class is imported at the top of the file.